app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+  def new
+  end
```

Comment:
For empty methods use single line syntax
```suggestion
  def new; end
```